liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -810,29 +808,6 @@ protected void doExecute() throws Exception {
         }
     }
 
-    private void processContainerParams() throws MojoExecutionException {
-        if (container) {
-            // this also sets the project property for use in DeployMojoSupport
-            setContainer(true);
-            return;
-        } else {
-            if (dockerfile != null) {
-                if (dockerfile.exists()) {
-                    setContainer(true);
-                    return;
-                } else {
-                    throw new MojoExecutionException("The file " + dockerfile + " used for dev mode option dockerfile does not exist."
```

Comment:
Do we perform this check anywhere else to see if the Dockerfile exists?

Reply:
I believe this is the only check. We will need to keep the check for if the Dockerfile specified by the user exists:
```
if (dockerfile != null && !dockerfile.exists()) {
    throw new MojoExecutionException(...)
}
```

Reply:
It is checked in ci.common https://github.com/OpenLiberty/ci.common/blob/master/src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java#L565

Reply:
I guess this really comes down to methodology. There is a check in DevUtil in ci.common right before `dockerfile` is used, which was intended to cover any strange default Dockerfile scenarios. It can cover user parameters as well if we want it to. It comes down to do we want to validate parameters earlier on in the Maven/Gradle side, or validate parameters as we go to use them.